config/database.yml

```diff
@@ -0,0 +1,76 @@
+# PostgreSQL. Versions 9.3 and up are supported.
+
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  # For details on connection pooling, see Rails configuration guide
+  # https://guides.rubyonrails.org/configuring.html#database-pooling
+  # pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: 12345
```

Comment:
- [ ] The password/username on this part is not required, if you don't add it the local installation for your project will take the default username/password on a local machine, this is important if your project will be installed on diff local config (as my computer, for example). 🐶